### ISSUE TITLE
XMP orientation support

### ISSUE THREAD

**Issue by Yankovsky** [danialfarid/ng-file-upload#1878]
Currently ngf-fix-orientation supports only EXIF metadata, but does not support XMP. XMP has orientation info too. Example image with XMP metadata below.
[Check attached image metadata](http://metapicz.com/#landing?imgsrc=https%3A%2F%2Fcloud.githubusercontent.com%2Fassets%2F72094%2F23901048%2Fac41322c-08cc-11e7-88a1-3e9031e8b86f.jpg).
![XMP-orientation-image](https://cloud.githubusercontent.com/assets/72094/23901048/ac41322c-08cc-11e7-88a1-3e9031e8b86f.jpg)
![orientation](http://s.csssr.ru/U02D248T6/2017-03-14-1547-1sm0ycabpr.png)

I want to make pr fixing this issue. Are you interested in such feature? Is it ok to use third party libs to read XMP metadata?

**Comment by danialfarid** [comment]
Yes PR's are welcome. If the code can be extracted for that part I would prefer that unless it is very lightweight library.
